tests/lib/rules/prefer-screen-queries.test.ts

```diff
@@ -4,9 +4,11 @@ import { ALL_QUERIES_COMBINATIONS } from '../../../lib/utils';
 
 const ruleTester = createRuleTester();
 
+const ALL_QUERIES_COMBINATIONS_PLUS_CONTAINER = [...ALL_QUERIES_COMBINATIONS, 'container']
```

Comment:
I think that it would be better to keep `ALL_QUERIES_COMBINATIONS` and then use container for its specific scenarios. As this would test as valid scenarios that do not match real tests. For instance, in the line 12 it would translate to

```ts
screen.container()
```

which is not RTL valid code. Same below for

```js
within(component).container()
```